stix2elevator/stix_stepper.py

```diff
@@ -208,6 +313,9 @@ def step_file(fn, validator_options, encoding="utf-8"):
 def main():
     stepper_arg_parser = _get_arg_parser()
     stepper_args = stepper_arg_parser.parse_args()
+    initialize_options()
+    set_option_value("missing_policy", "use-extensions")
```

Comment:
You could add another argument so users of the script can pass their option_value for missing policy. The "use-extensions" could be the default value if none is set. (similar to the suggestion below)

Reply:
I don't really want to give users the option to choose anything else.  Do you disagree?

Reply:
For the missing_policy not having other options, I would use the a prior statement presented for the stix2-validator. The old method of producing custom content is not invalid, just deprecated. A user should be allowed to select which one they want and defaulting to using extensions, its fine by me. Considering the old way was already supported by the code, I don't think it would be a problem to provide the option to the user.